src/templates/pages/patterns/heading-subheading/index.js

```diff
@@ -0,0 +1,63 @@
+import { h } from 'preact';
+import PatternLayout from '@layouts/pattern';
+import CodeHero from '../../example/heading-subheading/hero/code';
+import CodeCard from '../../example/heading-subheading/card/code';
+import { render } from 'preact-render-to-string';
+import PatternTitle from '@components/pattern-title';
+import { STATUS } from '@constants';
+
+export const title = 'Heading with subheading';
+
+export const status = STATUS.DEVELOPMENT;
+
+const ExpandableSearch = () => <PatternLayout>
```

Comment:
Should this be HeadingSubheading or similar, rather than ExpandableSearch?

Reply:
Indeed - can you tell which file I started from? :D